src/ATen/function_wrapper.py

```diff
@@ -479,11 +505,15 @@ def create_derived(backend_type_env, declarations):
     type_object_declarations = []
     type_object_definitions = []
 
+    is_cuda = 'CUDA' in backend_type_env['Backend']
+
     def replace_with_null(argument):
         return (argument['type'] == 'THGenerator*' and
                 backend_type_env['Backend'] == 'CUDA')
 
     def requires_checked_cast(argument):
+        if argument['type'] == 'IntList':
```

Comment:
Do you want to expose `SizedIntList` in `Declarations.yaml`. I'm concerned that by simplifying `requires_checked_cast` it would complicate all the binding code which would now have to check for `IntList` or `SizedIntList`

Reply:
We don't have to -- in several other places the type in the .cwrap files there is one type ('THTensor*'), and the type that shows up in Declarations.yaml is another. I believe there is a dict that handles those translations already.